.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+
+language: scala
+
+services:
+- docker
+
+cache:
+  directories:
+  - $HOME/.ivy2/cache
+  - $HOME/.sbt/boot/
+
+script:
+- mkdir -p ${HOME}/.sbt
+- docker run -v ${HOME}/.ivy2:/root/.ivy2 -v ${HOME}/.sbt:/root/.sbt -v ${PWD}:/usace-flood-geoprocessing
```

Comment:
Docker usage in MMW was mainly to replicate the production environment even further in CI. I don't know how we're running this application, but if we aren't using Docker, then it would make more sense to not use it in CI. That said, I don't know that we lose anything by using it.

If we do drop it, I think that we would need to be more explicit about the JVM and Scala versioned used in the Travis run. Within the container, it's the Java 7 JRE and Scala 2.10.5.